src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -378,7 +378,7 @@ public RemoteBuildEventUploadModeConverter() {
   public boolean experimentalGuardAgainstConcurrentChanges;
 
   @Option(
-      name = "experimental_remote_grpc_log",
+      name = "remote_grpc_log",
```

Comment:
Please set oldName to the previous name so it's easier to migrate.

Reply:
Updated, thanks!